packages/web-comment/src/routes/index.js

```diff
@@ -0,0 +1,38 @@
+const { readdirSync } = require('fs');
+
+// works line Next, urls come from dir names in routes director
+
+const dirNames = readdirSync(__dirname).filter((name) => /.*(?<!(\.js))$/.exec(name));
+
+const routes = dirNames.reduce((acc, dirName) => {
```

Comment:
Some jsdoc here so it's clear the return type would be great.

I think it's `Object<string, import('express').IRouter>`?

Reply:
yeah agreed